### PR TITLE
Documentation + Resources for Sylius 1.9

### PR DESCRIPTION
**Pull request by Snowbaha**
Hello!

Thank you for this plugin, it was what I was looking for! 

I started to use it with Sylius 1.9 and I had to had this so I share with you the fix :)

I hope it can help you.

### CODE REVIEW

**Comment by Snowbaha** [comment]
Hello, are you stopping the support of the plugin?

**Comment by Mauricette** [comment]
Hello, 
thanks for you submission and sorry for the delay, i ll check.

File: composer.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.2",
-        "sylius/sylius": "^1.4"
+        "sylius/sylius": "^1.9"
```
Comment (Mauricette) [inline composer.json]: It seems to work on sylius 1.6, is it really necessary to leave this requirement?

Comment (Snowbaha) [reply composer.json]: It's because I am on php7.4 with Sylius 1.9 so I cannot test it below but if it's ok, no problem ^^

Comment (Mauricette) [reply composer.json]: Perfect, can you leave 1.4 compatibility? After that I merge, thank you!

**Comment by Snowbaha** [comment]
Hello, can you add a news tag version when you can please :)

**Comment by loicsapone** [comment]
Hi :hand:

Thanks for your PR!
Sorry for the delay, I just tagged v0.3.0 (https://github.com/Mezcalito/SyliusFileUploadPlugin/releases/tag/0.3.0)

**Pull request merged**